dist/Tie-File/lib/Tie/File.pm

```diff
@@ -2128,6 +2129,16 @@ then the array C<@gifts> would appear to contain C<("Gold\n",
 "Frankincense\n", "Myrrh\n")>, or (on Win32 systems) C<("Gold\r\n",
 "Frankincense\r\n", "Myrrh\r\n")>.
 
+=head2 C<binmode>
+
+Specify the binmode to set for files opened by Tie::File.
+
+  # make sure Perl is expecting unicode files and strings
+  use utf8;
+  use feature 'unicode_strings';
```

Comment:
Your documentation for the `binmode` option states that to use it you have to ensure that the program "is expecting unicode file and strings", and that this is ensured by calling `use utf8; use feature 'unicode_strings';`.  However, the test program you've added to the repository, does not call `use feature 'unicode_strings';`.  Which is correct?  The documentation or the test program?

Reply:
Those 2 lines are recommended, one of the earlier comments requested I remove unicode_strings from the test because it isn't necessary for the test to work. I can make the doc clearer that these two lines are recommended.